jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -201,12 +204,14 @@ def _update_plugin_marks(self, *args):
         if self.setting_interactive_extract:
             try:
                 sp1d = self.create_extract(add_data=False)
-            except Exception:
+            except Exception as e:
                 # NOTE: ignore error, but will be raised when clicking ANY of the export buttons
                 # NOTE: KosmosTrace or manual background are often giving a
                 # "background regions overlapped" error from specreduce
+                self.ext_specreduce_err = str(e)
```

Comment:
Are you sure you just want the error message and not "ErrorClass(error message)"?

Reply:
I'm not sure, is that useful to a UI-user?  I guess it is for us to debug, but might be pretty meaningless otherwise.  Would you just suggest to switch to `repr(e)` here as well?

Reply:
For a dev, more info is always better. So my vote is a yes.